Minio.Tests/NegativeTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿/*
+ * MinIO .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2017 MinIO, Inc.
```

Comment:
We have to fix the YEAR here. In other places as well.

Reply:
Ok, will do

Reply:
Done:)

Reply:
@ig-sinicyn, the year needs to be fixed only on the files touched by your PR. Could you revert the changes to rest of the files?

Reply:
Done:)